lang/main.json

```diff
@@ -786,6 +786,7 @@
         "newDeviceAction": "Use",
         "newDeviceAudioTitle": "New audio device detected",
         "newDeviceCameraTitle": "New camera detected",
+        "nextToSpeak": "Your turn to speak might come soon, get ready",
```

Comment:
Let's do the next to speak as a separate PR, since it's not related to lowering the hand.